From eae0c456b1e23cb1b1f314b42eab8402b372ce25 Mon Sep 17 00:00:00 2001
From: Yetizone <andreii.lisita@gmail.com>
Date: Wed, 14 Oct 2020 19:48:00 +0300
Subject: [PATCH 07/18] Fix build warning

Replace Gtk.show_uri with Gtk.show_uri_on_window as suggested in
https://valadoc.org/gtk+-3.0/Gtk.show_uri.html
---
 src/window.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/window.vala b/src/window.vala
index 623b1d4..669f177 100644
--- a/src/window.vala
+++ b/src/window.vala
@@ -263,7 +263,7 @@ public class Window : Hdy.ApplicationWindow {
 
     private void on_help_activate () {
         try {
-            Gtk.show_uri (get_screen (), "help:gnome-clocks", Gtk.get_current_event_time ());
+            Gtk.show_uri_on_window (this, "help:gnome-clocks", Gtk.get_current_event_time ());
         } catch (Error e) {
             warning (_("Failed to show help: %s"), e.message);
         }
-- 
2.29.2

